### PR TITLE
fix(PhotoSyncSettingsViewController): This class no longer uses a storyboard to load

### PR DESCRIPTION
**Pull request by adrien-coye**
Some leftover XIB loading code was removed, and a crash on table view refresh was fixed also.

### CODE REVIEW

File: kDrive/AppRouter.swift [hunk 1]
```diff
@@ -397,7 +397,7 @@ public struct AppRouter: AppNavigable {
             return
         }
 
-        let photoSyncSettingsViewController = PhotoSyncSettingsViewController.instantiate()
+        let photoSyncSettingsViewController = PhotoSyncSettingsViewController()
```
Comment (adrien-coye) [inline kDrive/AppRouter.swift:400]: This is no longer loaded by XIB, therefore no more instantiate method

File: kDrive/UI/Controller/LaunchPanelsController.swift [hunk 2]
```diff
@@ -60,15 +60,9 @@ class LaunchPanelsController {
...
-                    let mainTabVC = UIApplication.shared.delegate?.window??.rootViewController as? MainTabViewController
-                    guard let currentVC = mainTabVC?.selectedViewController as? UINavigationController else {
-                        return
-                    }
-                    currentVC.dismiss(animated: true)
-                    currentVC.setInfomaniakAppearanceNavigationBar()
-                    currentVC.pushViewController(photoSyncSettingsVC, animated: true)
+                floatingPanelViewController?.actionHandler = { @MainActor _ in
+                    @InjectService var appNavigable: AppNavigable
+                    appNavigable.showPhotoSyncSettings()
```
Comment (adrien-coye) [inline kDrive/UI/Controller/LaunchPanelsController.swift:65]: We can use the `Router` for this 🎉

File: kDrive/UI/Controller/LaunchPanelsController.swift [hunk 3]
```diff
@@ -106,11 +100,13 @@ class LaunchPanelsController {
     /// Pick and display a panel, if any, on the specified view controller.
     /// - Parameter viewController: View controller to present the panel.
     func pickAndDisplayPanel(viewController: UIViewController) {
-        DispatchQueue.global().async {
-            if let panel = self.pickPanelToDisplay() {
-                Task { @MainActor in
-                    viewController.present(panel.makePanelController(), animated: true, completion: panel.onDisplay)
-                }
+        Task {
```
Comment (adrien-coye) [inline kDrive/UI/Controller/LaunchPanelsController.swift:103]: Removed some old GCD code and reworked with a guard.

File: kDrive/UI/Controller/Menu/PhotoSyncSettingsViewController.swift [hunk 4]
```diff
@@ -167,6 +167,7 @@ final class PhotoSyncSettingsViewController: BaseGroupedTableViewController {
         }
     }
 
+    /// This method mutates `sections`, must be called within a `performBatchUpdates`
```
Comment (adrien-coye) [inline kDrive/UI/Controller/Menu/PhotoSyncSettingsViewController.swift:170]: This was another crash I had, the comment explains it.

**Review by adrien-coye — COMMENTED** [review]
Left some notes for reviewer.

**Review by Ambrdctr — APPROVED** [review]

**Pull request merged**